src/tests/src/libs/antares/study/test_study.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+** Copyright 2007-2023 RTE
+** Authors: Antares_Simulator Team
+**
+** This file is part of Antares_Simulator.
+**
+** Antares_Simulator is free software: you can redistribute it and/or modify
+** it under the terms of the GNU General Public License as published by
+** the Free Software Foundation, either version 3 of the License, or
+** (at your option) any later version.
+**
+** There are special exceptions to the terms and conditions of the
+** license as they are applied to this software. View the full text of
+** the exceptions in file COPYING.txt in the directory of this software
+** distribution
+**
+** Antares_Simulator is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+** GNU General Public License for more details.
+**
+** You should have received a copy of the GNU General Public License
+** along with Antares_Simulator. If not, see <http://www.gnu.org/licenses/>.
+**
+** SPDX-License-Identifier: licenceRef-GPL3_WITH_RTE-Exceptions
+*/
+
+#define BOOST_TEST_MODULE study
+#define BOOST_TEST_DYN_LINK
+#define WIN32_LEAN_AND_MEAN
+#include <boost/test/unit_test.hpp>
+
+#include "antares/study/study.h"
+
+using namespace Antares::Data;
+
+BOOST_AUTO_TEST_CASE(basic_areas_operations)
```

Comment:
It's a good thing to test these functions, but they're only used in the **GUI**, not in the solver or anywhere else.
Do we want to invest on tests about **GUI** code ?

Reply:
As long as they are present in the lib we need to guarantee they work unfortunately.
I don't know when we will actually completely drop the GUI, this could save us some more regressions :) 

Note also that we may want to provide the possibility to modify studies through other interface than the GUI, later (like python bindings).